_drafts/community-group-overview.md

```diff
@@ -1,48 +1,55 @@
+To help stimulate interest in bioinformatics and data science while also encouraging learning and growth we here at [the Coop](https://research.fhcrc.org/coop/en.html) have started a [newsletter](https://research.fhcrc.org/coop/en/newsletter.html), manage [slack channels](https://fredhutch.github.io/coop/community/slack-new/), and publish a [calendar](https://fredhutch.github.io/FHBig/calendar/) of events. We've also established and actively support and encourage community groups. [Community groups](https://research.fhcrc.org/coop/en/community/hosted-groups.html) are loose associations of individuals in the community who come together regularly around a particular tool, language, skill etc. and meet either quarterly, monthly, or weekly depending on the groups needs. These meetings usually focus on best practices, new approaches, tools, information sharing, or demoing a new workflow or process that others in the community may find valuable.
 
-#### What they are
+#### Current Community Groups
 
-Here are some of the current community groups that we have here at the Hutch but it's not exhaustive as you can always create a new one but following the [guidelines here](.../2010-01-07-post-overview_community_group#how-to-start-one).
+Here are some of the [current community groups](https://research.fhcrc.org/coop/en/community/hosted-groups.html) that we have here at the Hutch but it's not exhaustive as you can always create a new one by following the guidelines below.
 
-- **Coop community meetings**: The Coop hosts community meetings once a quarter to share information from the data science community at the Hutch. Join us to learn about projects and events happening in our community! _**When & where**: Quarterly, usually A1-303 but check calendars_
+- **Coop Community Meetings**: The Coop hosts community meetings once a quarter to share information from the data science community at the Hutch. Join us to learn about projects and events happening in our community! When & where: Quarterly, usually A1-303 but check the [calendar](https://fredhutch.github.io/FHBig/calendar/).
```

Comment:
Do we need to say "check the calendar" for EVERY group, or can we list that once at the top?

Reply:
removed the multiple mentions and reference it once at the top and again in the Coop Monthly meeting blurb just bc that event changes locations sometimes...